src/app/inventory/store/d2-item-factory.service.js

```diff
@@ -379,7 +379,8 @@ export function D2ItemFactory(
     // Infusion
     const tier = itemDef.inventory ? defs.ItemTierType[itemDef.inventory.tierTypeHash] : null;
     createdItem.infusionProcess = tier && tier.infusionProcess;
-    createdItem.infusable = Boolean(createdItem.infusionProcess && itemDef.quality && itemDef.quality.infusionCategoryHashes && itemDef.quality.infusionCategoryHashes.length);
+    createdItem.infusionFuel = Boolean(createdItem.infusionProcess && itemDef.quality && itemDef.quality.infusionCategoryHashes && itemDef.quality.infusionCategoryHashes.length);
+    createdItem.infusable = createdItem.infusionFuel && isLegendaryOrBetter(createdItem);
```

Comment:
I'm not sure the `isLegendaryOrBetter` check is necessary - we're already looking at the tier to see if there's an infusion process, which I think is what defines which tiers are allowed to infuse. Am I understanding that right!

Reply:
It appears the Rare tier type has `infusionProcess` defined, but it isn't infusable in-game.
On master, this was causing the Fuel Finder button to appear in the item details popup for items where it didn't make sense.
Example from master:
<img width="414" alt="dim-reverse-infuse-live-bug" src="https://user-images.githubusercontent.com/6517289/34647501-3ea3e93c-f339-11e7-96a2-6fc9607d4a97.png">

In the new logic, we determine whether the item is eligible to be used as fuel or is infusable. This lets us specifically show the Fuel Finder button only for items where it makes sense.

Reply:
Good to know, thanks!